### ISSUE TITLE
How prepared do you think you are for the exam tomorrow?

### ISSUE THREAD

**Issue by leedosdos** [nus-cs2030/2122-s1#372]
Just a random thought whether there are people like me out there who don't feel very prepared :(
Also, how are you guys doing your final revision before the exam?

**Comment by yeehaoo** [comment]
Yeah I have no idea how to revise for this kind of mod. Especially since the format of this year's final doesn't even follow the previous years. So I just created a reference sheet for the concepts and like a short explanation for each of them (less than 1 page) and I'm just looking through the past year papers now although I'm not even sure if most of the stuff is applicable

**Comment by Bacon-Strips** [comment]
I think I'll be pushing y'all from the back of the bell curve.

**Comment by yeehaoo** [comment]
> I think I'll be pushing y'all from the back of the bell curve.

same here :(

**Comment by 9frazz** [comment]
i'm reading through our lecture slides and i don't think anything is going in :"(

**Comment by RussellDash332** [comment]
![image](https://user-images.githubusercontent.com/63991775/143230665-477f634b-49e4-4335-a615-2a6d1af8d65a.png)

**Comment by e0543700** [comment]
is it just me or are the past year papers looking kinda difficult?

**Comment by satsuma42** [comment]
> is it just me or are the past year papers looking kinda difficult?

they do but bear in mind the paper format changes drastically if you look at 17/18 and 20/21 papers specifically. maybe they just look unfamiliar instead of difficult(?)

**Comment by jerylseow** [comment]
same here. good luck everyone!

**Comment by e0543700** [comment]
> > is it just me or are the past year papers looking kinda difficult?
> 
> they do but bear in mind the paper format changes drastically if you look at 17/18 and 20/21 papers specifically. maybe they just look unfamiliar instead of difficult(?)

are they out of syllabus? or still the same syllabus just different format?

**Comment by Devvynneo** [comment]
> ![image](https://user-images.githubusercontent.com/63991775/143230665-477f634b-49e4-4335-a615-2a6d1af8d65a.png)

As what russell posted, I will tank the bellcurve for you all so yall can get your As

**Comment by BrendanCheong** [comment]
gone liao

**Comment by AngJunYang** [comment]
too tired after trying to fix project to revise ded now

**Comment by garfield-oo7** [comment]
> I think I'll be pushing y'all from the back of the bell curve.

Same 🥲

**Comment by Bacon-Strips** [comment]
> too tired after trying to fix project to revise ded now

Remind me to never try to last minute projects in the future.

**Comment by ryan195** [comment]
I don't even know how to run code on my IDE lmao gon die

**Comment by josiahoo** [comment]
feeling quite unprepared too tbh HAHAHA just try our best tomorrow i guess and don't stress too much about how other people are doing, focus on your own learning :-)

**Comment by UnwilledTangent** [comment]
I just hope I don't remod lol
